### ISSUE TITLE
TimedRollingFileTree no call super 

### ISSUE THREAD

**Issue by duzechao** [magillus/flutter-fimber#75]
```
 TimedRollingFileTree(
      {this.timeSpan = TimedRollingFileTree.dailyTime,
      logFormat = CustomFormatTree.defaultFormat,
      this.filenamePrefix = "log_",
      this.filenamePostfix = ".txt",
      logLevels = CustomFormatTree.defaultLevels}) {
    fileNameFormatter = LogFileNameFormatter.full(
        prefix: filenamePrefix, postfix: filenamePostfix);
    rollToNextFile();
  }
```

should like this
```
TimedRollingFileTree(
      {this.timeSpan = TimedRollingFileTree.dailyTime,
      logFormat = CustomFormatTree.defaultFormat,
      this.filenamePrefix = "log_",
      this.filenamePostfix = ".txt",
      logLevels = CustomFormatTree.defaultLevels}) :super(logLevels: logLevels, logFormat: logFormat){
    fileNameFormatter = LogFileNameFormatter.full(
        prefix: filenamePrefix, postfix: filenamePostfix);
    rollToNextFile();
  }
```
to call super(logLevels: logLevels, logFormat: logFormat)

**Comment by magillus** [comment]
Hi, Thank you for posting this issue. 
Will fix it at nearest future.

**Comment by magillus** [comment]
Resolved in 0.4.2
